<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>650</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>YM2608 Tone Editor</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QFrame" name="nameFrame">
        <property name="styleSheet">
         <string notr="true">QFrame#nameFrame {
	border: 1px solid  silver;
	border-radius: 4px;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="nameLabel">
           <property name="text">
            <string>name</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QPushButton" name="nameButton">
           <property name="text">
            <string>Edit name</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Bank</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0" colspan="2">
          <widget class="QListWidget" name="listWidget">
           <property name="dragEnabled">
            <bool>true</bool>
           </property>
           <property name="dragDropMode">
            <enum>QAbstractItemView::DragDrop</enum>
           </property>
           <property name="defaultDropAction">
            <enum>Qt::MoveAction</enum>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="removeTonePushButton">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="newTonePushButton">
           <property name="text">
            <string>New</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QLineEdit" name="lineEdit">
           <property name="placeholderText">
            <string>Search...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="verticalLine">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QGroupBox" name="lfoGroupBox">
        <property name="title">
         <string>LFO</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="LabeledHSlider" name="freqSlider" native="true"/>
         </item>
         <item>
          <widget class="LabeledHSlider" name="pmsSlider" native="true"/>
         </item>
         <item>
          <widget class="LabeledHSlider" name="amsSlider" native="true"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="LabeledHSlider" name="alSlider" native="true"/>
        </item>
        <item>
         <widget class="LabeledHSlider" name="fbSlider" native="true"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="OperatorSliders" name="op1Sliders" native="true"/>
        </item>
        <item>
         <widget class="OperatorSliders" name="op2Sliders" native="true"/>
        </item>
        <item>
         <widget class="OperatorSliders" name="op3Sliders" native="true"/>
        </item>
        <item>
         <widget class="OperatorSliders" name="op4Sliders" native="true"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen_O"/>
    <addaction name="actionSave_S"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionO_pen_Bank"/>
    <addaction name="actionSave_Bank_As"/>
    <addaction name="separator"/>
    <addaction name="actionOpe_n_Song"/>
    <addaction name="separator"/>
    <addaction name="actionConvert_To_Text_C"/>
    <addaction name="actionRead_Text_R"/>
    <addaction name="separator"/>
    <addaction name="actionSetup_E"/>
    <addaction name="separator"/>
    <addaction name="actionExit_X"/>
   </widget>
   <widget class="QMenu" name="menuHelp_H">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout_A"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp_H"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_O">
   <property name="text">
    <string>&amp;Open Tone...</string>
   </property>
   <property name="toolTip">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave_S">
   <property name="text">
    <string>&amp;Save Tone</string>
   </property>
   <property name="toolTip">
    <string>Save Tone</string>
   </property>
  </action>
  <action name="actionExit_X">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="toolTip">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save Tone &amp;As...</string>
   </property>
  </action>
  <action name="actionConvert_To_Text_C">
   <property name="text">
    <string>&amp;Convert To Text...</string>
   </property>
   <property name="toolTip">
    <string>Convert To Text</string>
   </property>
  </action>
  <action name="actionSetup_E">
   <property name="text">
    <string>S&amp;etup...</string>
   </property>
   <property name="toolTip">
    <string>Setup</string>
   </property>
  </action>
  <action name="actionAbout_A">
   <property name="text">
    <string>&amp;About...</string>
   </property>
   <property name="toolTip">
    <string>About</string>
   </property>
  </action>
  <action name="actionRead_Text_R">
   <property name="text">
    <string>&amp;Read Text...</string>
   </property>
   <property name="toolTip">
    <string>Read Text</string>
   </property>
  </action>
  <action name="actionSave_Bank_As">
   <property name="text">
    <string>Sa&amp;ve Bank As...</string>
   </property>
  </action>
  <action name="actionO_pen_Bank">
   <property name="text">
    <string>O&amp;pen Bank...</string>
   </property>
  </action>
  <action name="actionOpe_n_Song">
   <property name="text">
    <string>Ope&amp;n Song...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>OperatorSliders</class>
   <extends>QWidget</extends>
   <header>widgets/operator_sliders.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LabeledHSlider</class>
   <extends>QWidget</extends>
   <header>widgets/labeled_hslider.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>nameButton</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>listWidget</tabstop>
  <tabstop>newTonePushButton</tabstop>
  <tabstop>removeTonePushButton</tabstop>
  <tabstop>lfoGroupBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
